src/QuadForm/Quad/ZLattices.jl

```diff
@@ -2863,6 +2863,71 @@ function glue_map(L::ZZLat, S::ZZLat, R::ZZLat; check=true)
   return glue_map, iS, iR
 end
 
+@doc raw"""
+    primitive_extension(glue_map::TorQuadModuleMap)
```

Comment:
```suggestion
    primitive_extension(glue_map::TorQuadModuleMap) -> ZZLat, QQMatrix, QQMatrix
```

Reply:
Ah right, I forgot this type existed. Well if it is clear from the documentation what the output is, maybe we can omit it.